lib/checks/mobile/css-orientation-lock-evaluate.js

```diff
@@ -103,21 +103,31 @@ function cssOrientationLockEvaluate(node, options, virtualNode, context) {
 
     const transformStyle =
       style.transform || style.webkitTransform || style.msTransform || false;
-    if (!transformStyle) {
+    if (!transformStyle && !style.rotate) {
       return false;
     }
 
+    let transformFn;
+    let transformFnValue;
+
     /**
      * get last match/occurrence of a transformation function that can affect rotation along Z axis
      */
-    const matches = transformStyle.match(
-      /(rotate|rotateZ|rotate3d|matrix|matrix3d)\(([^)]+)\)(?!.*(rotate|rotateZ|rotate3d|matrix|matrix3d))/
-    );
-    if (!matches) {
-      return false;
+    if (transformStyle) {
```

Comment:
We should probably account for someone using both `transform` and `rotate`. I can well imagine someone using them together, for example using transform to adjust the position of an element and rotate to turn it.